components/MainActionBox.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Box } from "@primer/components";
+
+export const MainActionBox: React.FC = ({ children }) => (
+  <Box
+    display="flex"
+    flexDirection="column"
+    alignItems="center"
+    padding={8}
+    borderWidth={1}
+    borderStyle="solid"
+    borderColor="border.default"
+    borderRadius={16}
+    bg="canvas.subtle"
```

Comment:
are we going to use the `canvas.???` styles or the `bg.????` styles?

Reply:
Seems [from Primer Colors page](https://primer.style/primitives/colors) that the `bg.*` are removed/deprecated, and it's encouraged to use the `canvas.*` instead. Did I understand it right?

Reply:
See this section, [Removed variables](https://primer.style/primitives/colors#removed-variables)

Reply:
Ah my mistake! Any chance you can check for other mistaken uses of `bg.???` and change them to the relevant canvas colours?

Reply:
Yeah, I'll do!

Reply:
Did the `bg.*` replacement, and one additional thing: updated also `chats/[new].tsx` to use the common style box as in `login` & `invite`, so it looks more consistent.

<img width="1280" alt="Screenshot 2021-10-04 at 13 59 25" src="https://user-images.githubusercontent.com/1535759/135840185-9b6b6987-b384-4c9e-bd37-a096fae5c313.png">

Reply:
Awesome, will look over and should be good to merge